docs/configuration.md

```diff
@@ -72,7 +72,7 @@ Also, occassionally you will have a case where you have to supress the warnings
 That is fine, there are good exceptions to these recommendations.
 Don't be fundamentalistic about them.
 
-#### `enforceActions`
+#### `enforceActions: "observed" | "always" | "never" = "observed"`
```

Comment:
Could you put the possible values in the text, something like:

```
Usage: `enforceActions: "observed" | "always" etc
```

Otherwise any link to this config item will stop working, and it will be hard to maintain links if the possible values change over time

Reply:
Hm that didn't occur to me, what about:
`### [enforceActions: "observed" | "always" | "never" = "observed"](#enforceActions)`